MASH-FRET/source/simulations/plotExample.m

```diff
@@ -283,22 +283,35 @@ function plotExample(h_fig)
 %     Pdark = @(nic,noe,nie,g,f,r) (P(nie,G(noe,nie,g))*N(f*nic,noe,r))*(f*nic);      
 
     case 'poiss' % Poisson or P-model from B�rner et al. 2017
-        % convert camera offset in photon counts
-        I_th = arb2phtn(noisePrm(1));
```

Comment:
We might need to update noisePrm from CamNoise parameter field.

Reply:
You mean to rename the variable to CamNoise?

Reply:
We implement this comment in the next commit.